_includes/configuration/icons.md

```diff
@@ -0,0 +1,15 @@
+## icons
+
+**icons** documentations goes here
```

Comment:
uh, this seems incomplete?

Reply:
Yeah, someone more techical skills must help out here...

Reply:
I've delegated this to Simon

Reply:
Simon has updated this